docs/meetings/2019-11-22.md

```diff
@@ -0,0 +1,62 @@
+---
+id: meetings_20191122
```

Comment:
I would just record meeting. there is no way that we gonna document them properly.